release-pipeline.yml

```diff
@@ -860,6 +962,13 @@ stages:
               archiveFile: '$(archives)/azcopy_windows_386_$(azcopy_version).zip'
             continueOnError: true
 
+          - task: ArchiveFiles@2
+            displayName: 'Archive Windows 32 bit Build'
+            inputs:
+              rootFolderOrFile: '$(windows_386)'
```

Comment:
This will override the existing zip file created in previous step. You will have to use $(windows_grpc_386)